ecs-logs-collector.sh

```diff
@@ -496,6 +499,58 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/docker-sysconfig
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+  fi
+
+ try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker-storage "${info_system}"/docker/docker-sysconfig-storage
```

Comment:
nit: I think most of the other files are prefixed to gather them together, making this `sysconfig-docker-storage` instead. This also more closely resembles the files that folks might be chasing after and find more easily this way.

Reply:
Done!